### ISSUE TITLE
admin's dashboard gets solr error when new app with no repository objects

### ISSUE THREAD

**Issue by elrayle** [samvera/hyrax#2844]
### Descriptive summary

A brand new app with no repository objects gets a solr error when trying to display the dashboard for admin users.  It works fine for non-admin users.

### Rationale

Core functionality should work out of the box.

### Expected behavior

Dashboard should show metrics as empty when there are 0 repository objects.

### Actual behavior

When displaying the dashboard for admin users when 0 repository objects, throws an exception...

```
RSolr::Error::Http in Hyrax::Dashboard#show

Showing /Users/elr37/.rvm/gems/ruby-2.4.1/bundler/gems/hyrax-5dd570b0afb7/app/views/hyrax/dashboard/show_admin.html.erb where line #66 raised:

RSolr::Error::Http - 400 Bad Request
Error: {
  "responseHeader":{
    "status":400,
    "QTime":1,
    "params":{
      "q":"() AND (system_create_dtsi:[2017-12-25T00:00:00Z TO 2018-01-01T00:00:00Z])",
      "qt":"standard",
      "rows":"0",
      "wt":"json"}},
  "error":{
    "metadata":[

URI: http://127.0.0.1:8983/solr/hydra-development/select?wt=json&rows=0&q=%28%29+AND+%28system_create_dtsi%3A%5B2017-12-25T00%3A00%3A00Z+TO+2018-01-01T00%3A00%3A00Z%5D%29&qt=standard

Backtrace: /Users/elr37/.rvm/gems/ruby-2.4.1/gems/rsolr-2.1.0/lib/rsolr/client.rb:196:in `rescue in execute'
/Users/elr37/.rvm/gems/ruby-2.4.1/gems/rsolr-2.1.0/lib/rsolr/client.rb:186:in `execute'
/Users/elr37/.rvm/gems/ruby-2.4.1/gems/rsolr-2.1.0/lib/rsolr/client.rb:181:in `send_and_receive'
(eval):2:in `get'
/Users/elr37/.rvm/gems/ruby-2.4.1/gems/active-fedora-11.5.2/lib/active_fedora/solr_service.rb:44:in `get'
/Users/elr37/.rvm/gems/ruby-2.4.1/gems/active-fedora-11.5.2/lib/active_fedora/solr_service.rb:79:in `count'
/Users/elr37/.rvm/gems/ruby-2.4.1/gems/active-fedora-11.5.2/lib/active_fedora/relation/calculations.rb:11:in `count'
/Users/elr37/.rvm/gems/ruby-2.4.1/bundler/gems/hyrax-5dd570b0afb7/app/services/hyrax/statistics/over_time.rb:34:in `point'
/Users/elr37/.rvm/gems/ruby-2.4.1/bundler/gems/hyrax-5dd570b0afb7/app/services/hyrax/statistics/over_time.rb:26:in `block in points'
/Users/elr37/.rvm/gems/ruby-2.4.1/bundler/gems/hyrax-5dd570b0afb7/app/presenters/hyrax/admin/repository_growth_presenter.rb:10:in `each'
/Users/elr37/.rvm/gems/ruby-2.4.1/bundler/gems/hyrax-5dd570b0afb7/app/presenters/hyrax/admin/repository_growth_presenter.rb:10:in `each'
```

### Steps to reproduce the behavior

1. Create a new Hyrax 2.1.0 app
1. Login as an admin (which forwards you to the dashboard)

The dashboard should display with widgets displaying an appropriate value for 0 repository objects.

**Comment by elrayle** [comment]
@vantuyls Do you want to confirm that this one is a blocker?

**Comment by revgum** [comment]
@elrayle sanity check; I used v2.1.0.beta1c_qa branch, regenerated the internal test app, generated a user, gave her admin in role_map.yml, fcrepo_wrapper'd, solr_wrapper'd, and ran the rails server.. it *looks* like it's working. 

<img width="1252" alt="screen shot 2018-03-27 at 2 18 35 pm" src="https://user-images.githubusercontent.com/32885/37995716-33a68b3c-31ca-11e8-84c0-4f65ff61b2d5.png">

**Comment by vantuyls** [comment]
blocker if true, but maybe we need more testing?

**Comment by revgum** [comment]
Verified that this is broken when using the steps for standing up a new Hyrax application (rather than when using it in the context of engine_cart:generate)

**Comment by elrayle** [comment]
Fixed by PR #2860
